tests/end_to_end_cases/operations_api.rs

```diff
@@ -0,0 +1,92 @@
+use crate::common::server_fixture::ServerFixture;
+use generated_types::google::protobuf::Any;
+use influxdb_iox_client::{
+    management::{self, generated_types::*},
+    operations, protobuf_type_url_eq,
+};
+use std::time::Duration;
+
+fn get_operation_metadata(metadata: Option<Any>) -> OperationMetadata {
+    assert!(metadata.is_some());
+    let metadata = metadata.unwrap();
+    assert!(protobuf_type_url_eq(&metadata.type_url, OPERATION_METADATA));
+    prost::Message::decode(metadata.value).expect("failed to decode metadata")
+}
+
+#[tokio::test]
+async fn test_operations() {
+    let server_fixture = ServerFixture::create_single_use().await;
+    let mut management_client = management::Client::new(server_fixture.grpc_channel());
+    let mut operations_client = operations::Client::new(server_fixture.grpc_channel());
```

Comment:
```suggestion
    let mut management_client =server_fixture.management_client();
    let mut operations_client = operations::Client::new(server_fixture.grpc_channel());
```
You can probably do something similar for operations_client but also we could do that as a follow on PR

Reply:
Aah yes - this wasn't in at the time I wrote this